tests/functional/spec/piping.spec.js

```diff
@@ -0,0 +1,36 @@
+const helpers = require('../helpers');
+const PipingAddressPage = require('../pages/surveys/piping/what-is-your-address.page');
+const PipingPersonPage = require('../pages/surveys/piping/household-composition.page');
+const MultiplePipingPage = require('../pages/surveys/piping/term-time-location.page');
+
+describe('Multiple piping into question and answer', function() {
+
+    var piping_schema = 'test_multiple_piping.json';
+
+  it('Given into multiple address fields, when i navigate to term-time-location question, I should see all values separated  by commas and spaces', function() {
+    return helpers.openQuestionnaire(piping_schema).then(() => {
```

Comment:
Instead of doing this for each scenario you can do a beforeEach to load the survey for each scenario instead e.g.` beforeEach('load the survey', function() {
return helpers.openQuestionnaire(piping_schema);}
);`